src/main/java/sda/training/App.java

```diff
@@ -1,55 +1,40 @@
 package sda.training;
 
-import org.hibernate.Session;
-import org.hibernate.SessionFactory;
-import sda.training.learning.Car;
-
 import java.util.Scanner;
 
-public class App
-{
-    public static void main( String[] args )
-    {
-        System.out.println( "Hello World!" );
-
-        HibernateFactory hibernateFactory = new HibernateFactory();
-        SessionFactory sessionFactory = hibernateFactory.getSessionFactory();
-        Session session = sessionFactory.openSession();
-
-        Scanner scanner = new Scanner(System.in);
-        System.out.println("WYBIERZ GRE: ");
-        System.out.println("1. Tu wstaw gre 🚀🚀🚀");
-        System.out.println("2. Tu wstaw gre 🚀🚀🚀");
-        System.out.println("3. Tu wstaw gre 🚀🚀🚀");
-        System.out.println("4. Tu wstaw gre 🚀🚀🚀");
-        int gameNumber = scanner.nextInt();
-
-        switch (gameNumber){
-            case 1:
-                System.out.println("Zaimplementuj mnie 🤣");
-                break;
-            case 2:
-                System.out.println("Zaimplementuj mnie 🤣");
-                break;
-            case 3:
-                System.out.println("Zaimplementuj mnie 🤣");
-                break;
-            case 4:
-                System.out.println("Zaimplementuj mnie 🤣");
-                break;
-            default:
-                System.out.println("Bardzo sie starałeś, lecz gry nie wybrałeś");
+public class App {
+    public static void main(String[] args) {
+
+        while (true) {
+            Scanner scanner = new Scanner(System.in);
+            System.out.println("WYBIERZ GRE: ");
+            System.out.println("1. Kamień-Papier-Nożyczki");
+            System.out.println("2. Tu wstaw gre 🚀🚀🚀");
+            System.out.println("3. Tu wstaw gre 🚀🚀🚀");
+            System.out.println("4. Tu wstaw gre 🚀🚀🚀");
+            System.out.println("5. Wyjdź z aplikacji");
+
+            int gameNumber = scanner.nextInt();
+            switch (gameNumber) {
+                case 1:
+                    sda.training.rps.app.App app = new sda.training.rps.app.App();
```

Comment:
a moze inna nazwa niz App ? wtedy ladniej by to wygladalo :D poza tym nie mam pojecia co robi to App, nazwa jest dosc malo mowi :)

Reply:
Zmienione :)